src/IB/IBFEMethod.cpp

```diff
@@ -226,15 +231,413 @@ get_x_and_FF(libMesh::VectorValue<double>& x,
     return;
 }
 
-static const Real PENALTY = 1.e10;
+void
+assemble_cg_diffusion(EquationSystems& es, const std::string& system_name)
```

Comment:
Possibly there are other implementations that could benefit from these assembly routines. Would it make sense to move them to a separate libMesh utility class (e.g. in IBTK)?

Reply:
yeah it could be nice to do this.